### PR TITLE
Implemented day-forecast page. Fixed mobile version of landing page.

### PR DESCRIPTION
**Pull request by Khris951**

### CODE REVIEW

File: weather-app/src/app/day-forecast/day-forecast.component.html [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    <div class="main-board">
+        <div class="title-table">
+            <span class="day-time">{{currentDay}}, {{currentTime}}</span>
+            <span class="location">{{currentCity}}, {{currentCountry}}</span>
+            <span><img src='../../assets/images/icons/star-dark-checked.svg' alt="star"></span>
+        </div>
+    <table>
+        <tr class="border">
+          <th class="border-right"></th>
+          <th class="border-right">0:00</th>
```
Comment (chifa) [inline weather-app/src/app/day-forecast/day-forecast.component.html:11]: I'd suggest to move as many weather parameters as possible to mocked variables in .ts (arrays of values for example)
To display them here with ngFor.

**Review by OlexiyPokormyakho — APPROVED** [review]

**Review by AlexBelousUa — APPROVED** [review]

File: weather-app/src/app/day-forecast/day-forecast.component.scss [hunk 2]
```diff
@@ -0,0 +1,195 @@
+@import "../../styles/styles.scss";
+@import "../../styles/variables";
+
+.main-board {
+    width: 80%;
+    height: 5%;
+    // display: block;
```
[the remark this answers is not in the document]
Comment (Khris951) [reply weather-app/src/app/day-forecast/day-forecast.component.scss]: Fixed.

File: weather-app/src/app/day-forecast/day-forecast.component.scss [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+            .title {
+                background-color: $dark-additional-background-color;
+                border-radius: 20px 20px 0 0;
+    
+                .main-info {
+                    color: $dark-text-color;
+                }
+            }
+
+            // .table {
```
[the remark this answers is not in the document]
Comment (Khris951) [reply weather-app/src/app/day-forecast/day-forecast.component.scss]: Fixed.

File: weather-app/src/app/day-forecast/day-forecast.component.html (hunk 1 above)
Comment (Khris951) [reply weather-app/src/app/day-forecast/day-forecast.component.html]: Fixed.

**Review by chifa — APPROVED** [review]

**Pull request merged**